src/DAL/migration/customTypes.sql

```diff
@@ -2,16 +2,16 @@
 
 CREATE TYPE "polygon_parts".insert_part_record AS
 (
-	"recordId" uuid,
+	"record_id" uuid,
```

Comment:
Please update in 'mc-models' all of the column's attributes.
 for example 'imagingTimeBeginUTC' need to be changed to 'imaging_time_begin_UTC' https://github.com/MapColonies/mc-models/blob/master/src/models/polygonParts/polygonPartRecord.ts#L89